packages/ui/src/components/va-form/VaFormNew.demo.vue

```diff
@@ -1,108 +0,0 @@
-<template>
```

Comment:
Name `FormFormItemInteraction` and please keep it. This story tests all form components work all right with form.

Let's mark it as stale, as it should have better code (maybe some reuse from component stories and form stories). It's not mission critical though as it's fine to develop.